### ISSUE TITLE
toolbars_below_content.css disables left-click menu inside the urlbar

### ISSUE THREAD

**Issue by mimteatr** [MrOtherGuy/firefox-csshacks#144]
Hello,

"toolbars_below_content.css" disables left-click menu inside the urlbar; nothing appears, can only select whatever text or link inside the urlbar but can't perform any other actions which usually are available (copy, delete, paste and go, etc). and trying to click several times in order to see if it would show crashed FF a few times.

I'm using F 93.0b6 (64-bit)

**Comment by MrOtherGuy** [comment]
Yeah, I can reproduce that. Unfortunately I don't think there's anything I can do to really fix this.

A workaround would be to first hit ESC to close the results popup and then open context menu which should then work. Context menu should close the results anyway so behavior should be the same - unfortunately you can't use CSS to close the results automatically.

**Comment by mimteatr** [comment]
I understand. Although I don't really like having to use the keyboard (specifically here to hit ESC) I think I can live with it lol. Thanks!

**Comment by mimteatr** [comment]
Hey, I know I closed this but I'm just stumbling upon another problem connected with the same css.

When I start typing something in the urlbar, the URL suggestions box popup is completely transparent! 

I tried disabling other css as well as disabling userscripts, but it came down tothat that this is the problematic css, when disabled, and url bar and tab bar are back on the top, both problems disappeared!

**Comment by MrOtherGuy** [comment]
Unfortunately I cannot reproduce the problem you describe. The urlbar results popup seems to have the correct background-color that your theme sets and that works correctly on both my Windows10 system and linux (Fedora 34).

If your theme sets urlbar background to transparent then that would certainly be transparent, but I doubt that's what you are experiencing. Still, I can't really do much else unless you can figure out why the background is transparent for you.

You could use [browser toolbox](https://developer.mozilla.org/en-US/docs/Tools/Browser_Toolbox) to inspect the UI, find `#urlbar-background` and see what its computed background is when urlbar is selected.

The context menu will not show up normally though. If urlbar is focused, you'll first need to hit ESC to close the popup and then context menu should work properly. Kinda unfortunate but at least it sort-of works.

**Comment by mimteatr** [comment]
Strangely the transparency problem disappeared after I've update FF to 93.0b9 (64-bit) today so I suppose it was rather a FF problem and not the css!

Btw, I'm using the default theme just in case.

Unfortunately yet, the context menu problem still persists, still using the ESC.

**Comment by mimteatr** [comment]
Hello my friend,

Two new issues with this css; 

1. right-clicking on any tab crashes the browser.

2. interacting with the urlbar freezes the browser; I'm only able to close or restart it.

I've disabled it for the moment, hoping these will be solved somehow, sometiime 🙏😊

Thanks!

**Comment by MrOtherGuy** [comment]
Well, that sounds bad to put it mildly.

Unfortunately I cannot reproduce either issue - I tried Firefox 93 and 94 on Windows. And I tried 95 on Fedora Linux (93 isn't available yet). So unfortunately I can't really do anything about this unless you or someone else who experiences these issues can figure out what causes them. Something OS specific would be my guess.
